data/locale/German.json

```diff
@@ -1,186 +1,189 @@
+    "from MCP": "von MCP",
+    "Error opening USB device": "Fehler beim Öffnen des USB-Laufwerks",
+    "Error opening internal memory": "Fehler beim Öffnen des NAND",
+    "Error starting async installation of": "Fehler beim Starten der Installation von",
     "Installation failed!": "Installation fehlgeschlagen!",
-    "Install the main game to the same storage medium first": "Installiere zuerst das Hauptspiel zum selben Laufwerk",
-    "Also make sure there is no error with the USB drive": "Stelle auch sicher das es keine Probleme mit dem USB Laufwerk gibt",
-    "Possible USB error": "Möglicherweise ein USB Fehler",
-    "Possible missing or bad title.tik file": "Möglicherweise fehlende oder fehlerhafte title.tik Datei",
-    "Possible incorrect console for DLC title.tik file": "Möglicherweise falsche Konsole für DLC title.tik Datei",
-    "Not enough free space on target device": "Nicht genug Platz auf dem Ziellaufwerk",
-    "Files might be corrupt or bad storage medium.\nTry redownloading files or reformat/replace target device": "Möglicherweise fehlerhafte Dateien oder schlechtes Ziellaufwerk\nVersuche nochmals runterzuladen oder formatiere/ersetze das Laufwerk",
-    "Possible USB failure. Check your drives power source.": "Möglicher USB Fehler. Prüfe deine Stromversorgung.",
-    "Files might be corrupt": "Dateien könnten fehlerhaft sein",
+    "Install the main game to the same storage medium first": "Installier zuerst das Hauptspiel auf demselben Datenträger",
+    "Also make sure there is no error with the USB drive": "Vergewisser dich auch, dass kein Fehler mit dem USB-Laufwerk vorliegt",
+    "Possible USB error": "Möglicherweise ein USB-Fehler",
+    "Possible missing or bad title.tik file": "Möglicherweise fehlende/fehlerhafte title.tik-Datei",
+    "Possible incorrect console for DLC title.tik file": "Möglicherweise falsche Konsole für die title.tik-DLC-Datei",
+    "Not enough free space on target device": "Nicht genügend Speicherplatz auf dem ausgewählten Datenträger verfügbar",
+    "Files might be corrupt or bad storage medium.\nTry redownloading files or reformat/replace target device": "Möglicherweise sind die Dateien oder das Speichermedium beschädigt.\nVersuch, die Dateien erneut herunterzuladen oder formatiere/ersetze den Datenträger",
+    "Possible USB failure. Check your drives power source.": "Möglicherweise ein USB-Fehler. Überprüf die Stromversorgung deines Laufwerks.",
+    "Files might be corrupt": "Dateien könnten beschädigt sein",
     "Unknown Error": "Unbekannter Fehler",
 
     "Network error": "Netzwerkfehler",
-    "check the network settings and try again": "prüfe die Netzwerkeinstellungen und versuche es nochmal",
-    "SSL error": "SSL Fehler",
-    "check your Wii Us date and time settings": "Prüfe die Datums- und Zeiteinstellungen deiner Wii U",
+    "check the network settings and try again": "Überprüf die Netzwerkeinstellungen und versuch es erneut",
```

Comment:
`versuch` is not a german word. It's either `versuche` (von `versuchen` or `der Versuch`, see:
https://www.duden.de/rechtschreibung/versuchen
https://www.duden.de/rechtschreibung/Versuch
Now granted the Duden doesn't explicy state `versuche` and I can't remember the grammar rule behind it but just google for `versuche es erneut` and you'll see that it's used everywhere.

`überprüf` also is not a german word. It's `überprüfe`. Same goes for the `Prüf` one line above. granted, you could do both as `Überprüf'` / `Prüf'` but that just looks completely unprofessional.